orbit/pkg/update/update.go

```diff
@@ -384,6 +384,12 @@ func (u *Updater) get(target string) (*LocalTarget, error) {
 					return nil, fmt.Errorf("failed to remove old extracted dir: %q: %w", localTarget.DirPath, err)
 				}
 			}
+			if strings.HasSuffix(localTarget.Path, ".pkg") {
```

Comment:
Double-checking: Should we confiirm if runtime is darwin before this?

Reply:
that's a good idea. I don't think this will affect anyone right now (you can't build an installer with Escrow Buddy, and the only `pkg` target we have is macOS) but I will add the check for the future.